src/mapping/AxialGeoMultiscaleMapping.cpp

```diff
@@ -0,0 +1,181 @@
+#include "AxialGeoMultiscaleMapping.hpp"
+#include <Eigen/src/Core/Matrix.h>
```

Comment:
```suggestion
#include<Eigen/Core>
```

This is dangerous, and I am also constantly running into this autocompletion mess too. We need a pre-commit check for this.